examples/04-grav/plot_inversion_linear.py

```diff
@@ -23,6 +23,16 @@
 
 def run(plotIt=True):
 
+    """
```

Comment:
@fourndo : I think the reason this example is failing is that the first docstring, with the title, now needs to be at the top of the file so that the sphinx-gallery can attach a title to it (have a look at  the examples in basic)